db-service/lib/common/DatabaseService.js

```diff
@@ -3,15 +3,183 @@ const ConnectionPool = require('./generic-pool')
 const infer = require('../infer')
 const cds = require('@sap/cds')
 
+const databaseModel = cds.linked({
+  definitions: {
+    schemas: new cds.entity({
```

Comment:
maybe a more distinct name, like `cds_test_schemas` to avoid clashes?

Reply:
The model is inside the `database` isolation. It actually clashes with `SYS.SCHEMAS` in `HANA`, but as this is isolated it doesn't matter.

- SYS
  - Database 1 <- database model
     - Tenant 1 <- application model
     - ...
   - ...